### PR TITLE
doc: Added citation subsection  in README

### PR DESCRIPTION
**Pull request by poudel-bibek**
Screenshot 2024-06-08 at 9 00 59 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,3 +129,16 @@ Distributed under the MIT License. See [`LICENSE`](./LICENSE) for more informati
 [issues-url]: https://github.com/OpenDevin/OpenDevin/issues
 [license-shield]: https://img.shields.io/github/license/opendevin/opendevin?style=for-the-badge
 [license-url]: https://github.com/OpenDevin/OpenDevin/blob/main/LICENSE
+
+## 📚 Cite
+
+```
+@misc{opendevin2024,
+  author       = {{OpenDevin Team}},
+  title        = {OpenDevin},
```
Comment (xingyaoww) [inline README.md:138]:
```suggestion
  title        = {{OpenDevin: An Open Platform for AI Software Developers as Generalist Agents}},
```

**Review by xingyaoww — COMMENTED** [review]
LGTM - After changing the title to match our technical report to help track it should be good!

**Comment by poudel-bibek** [comment]
Changed. Thanks!

Screenshot 2024-06-09 at 8 47 44 AM

**Review by xingyaoww — APPROVED** [review]
LGTM now!

**Pull request merged**